apps/src/studio/studio.js

```diff
@@ -1709,6 +1709,14 @@ Studio.reset = function(first) {
   Studio.sayComplete = 0;
   Studio.playSoundCount = 0;
 
+  // More things used to validate level completion.
+  Studio.trackedBehavior = {
```

Comment:
This is in Studio.reset, so it gets reset each time the simulation begins, doesn't it?